### PR TITLE
Introduce main for runtime-manager

### PR DESCRIPTION
**Pull request by honpey**
Signed-off-by: honpey <honpey@gmail.com>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -104,6 +104,10 @@ build-koord-manager: ## Build koord-manager binary.
 build-koord-scheduler: ## Build koord-scheduler binary.
 	go build -o bin/koord-scheduler cmd/koord-scheduler/main.go
 
+.PHONY: build-runtime-manager
```
Comment (jasonliu747) [inline Makefile:107]: can you add this step into the ci workflow?
https://github.com/koordinator-sh/koordinator/blob/ea8496c485835dac30f110c19c9270eff8c5c079/.github/workflows/ci.yaml#L90-L91

Comment (honpey) [reply Makefile:107]: done

File: cmd/runtime-manager/main.go [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
+	pflag.Parse()
+
+	switch options.BackendRuntimeMode {
+	case options.BackendRuntimeModeContainerd:
+		server := cri.NewRuntimeManagerCriServer()
+		go server.Run()
+	case options.BackendRuntimeModeDocker:
+	default:
+		klog.Errorf("unknown runtime engine backend %v", options.BackendRuntimeMode)
```
Comment (jasonliu747) [inline cmd/runtime-manager/main.go:50]: let's use `klog.Fatalf`.

Comment (honpey) [reply cmd/runtime-manager/main.go]: > let's use `klog.Fatalf`.

fixed

**Review by jasonliu747 — COMMENTED** [review]
/lgtm

**Pull request merged**